blazor/dropdown-list/value-binding.md

```diff
@@ -92,7 +94,7 @@ In the following example, `Game` column has been mapped to the [`DropDownListFie
 
 ## Enum binding
 
-You can bind enum data to DropDownList component. The following code helps you get a string value from the enumeration data.
+You can bind enum data to [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of DropDownList component. The following code helps you get a string value from the enumeration data.

```

Comment:
Bind the enum data to the [@bind-Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) attribute of the DropDownList component. The following code helps you to get a string value from the enumeration data.